src/keys/ecdh-browser.js

```diff
@@ -96,7 +97,7 @@ function unmarshalPublicKey (curve, key) {
   const byteLen = curveLengths[curve]
 
   if (!key.slice(0, 1).equals(Buffer.from([4]))) {
-    throw new Error('Invalid key format')
+    throw errcode('Cannot unmarshal public key - invalid key format', 'ERR_INVALID_KEY_FORMAT')
```

Comment:
use `new Error()`. This also happens in other places below, I will not point them out in this review